src/common/syncjournaldb.h

```diff
@@ -70,6 +70,9 @@ class OCSYNC_EXPORT SyncJournalDb : public QObject
     [[nodiscard]] bool getFilesBelowPath(const QByteArray &path, const std::function<void(const SyncJournalFileRecord&)> &rowCallback);
     [[nodiscard]] bool listFilesInPath(const QByteArray &path, const std::function<void(const SyncJournalFileRecord&)> &rowCallback);
     [[nodiscard]] Result<void, QString> setFileRecord(const SyncJournalFileRecord &record);
+    [[nodiscard]] bool getRootE2eFolderRecord(const QString &remoteFolderPath, SyncJournalFileRecord *rec);
+    [[nodiscard]] bool listAllE2eeFoldersWithEncryptionStatusLessThan(const int status, const std::function<void(const SyncJournalFileRecord &)> &rowCallback);
```

Comment:
same question from another place, I do not get why we are comparing with less or more for a status that is either encrypted or unencrypted (i.e. a boolean)

Reply:
@mgallien I don't know why, but when working on `v1.2` you called this a status, not a `version` or something numeric, I decided to keep it, do you have other ideas? https://github.com/nextcloud/desktop/blame/master/src/libsync/syncfileitem.h#L286